source/agora/node/FullNode.d

```diff
@@ -534,6 +535,24 @@ public class FullNode : API
                 config.node.block_interval_sec.seconds);
     }
 
+    /***************************************************************************
+
+        Get an enrolled UTXO for an address
+
+        Params:
+            pubkey = an address to find an UTXO with
+
+        Returns:
+            An UTXO if there is a validator for the public key,
+            otherwise UTXO.init
+
+    ***************************************************************************/
+
+    public UTXO getValidatorUTXO (in PublicKey pubkey) @trusted nothrow
+    {
+        assert(0);
```

Comment:
just a small thing: can we have assert(0, "some string here") - when we will switch to release build we might not have access to the stack trace, so we might not know what the actual error was

Reply:
Updated. There was a mistake. The code in `Validator` was in fact moved into this function because the `Validator` doesn't need to have it. Ty.